From a2d7a533251707cf1d5f6a6af448e44e19a6019a Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Thu, 30 Apr 2026 09:21:33 -0500
Subject: [PATCH] Add quotes to Maintainer field

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 6a4438dece07..ac54feeca677 100644
--- a/debian/control
+++ b/debian/control
@@ -1,7 +1,7 @@
 Source: ros-lyrical-apex-test-tools
 Section: misc
 Priority: optional
-Maintainer: Apex.AI, Inc. <opensource@apex.ai>
+Maintainer: "Apex.AI, Inc." <opensource@apex.ai>
 Build-Depends: debhelper (>= 9.0.0), ros-lyrical-ament-cmake, ros-lyrical-ament-cmake-auto, ros-lyrical-ament-cmake-gtest, ros-lyrical-ament-cmake-pclint <!nocheck>, ros-lyrical-ament-lint-auto <!nocheck>, ros-lyrical-ament-lint-common <!nocheck>, ros-lyrical-osrf-testing-tools-cpp, ros-lyrical-ros-workspace
 Homepage: https://index.ros.org/p/apex_test_tools/#lyrical
 Standards-Version: 3.9.2
-- 
2.53.0

